playbooks/robusta_playbooks/prometheus_enrichments.py

```diff
@@ -42,6 +47,52 @@ def parse_duration(
     return None, None
 
 
+@action
+def prometheus_all_available_metrics(event: ExecutionBaseEvent, prometheus_params: PrometheusParams):
+    result = get_prometheus_all_available_metrics(prometheus_params=prometheus_params)
+
+    event.add_enrichment([JsonBlock(json.dumps({"metrics": result}))])
+
+
+def get_prometheus_all_available_metrics(prometheus_params: PrometheusParams) -> List[str]:
+    try:
+        prom = get_prometheus_connect(prometheus_params=prometheus_params)
+        return prom.all_metrics()
+
+    except Exception as e:
+        raise Exception(f"An error occurred while fetching all available Prometheus metrics: {e}")
```

Comment:
If you're raising an error this way, you lose the original stack trace
I suggest logging the error, with exc_info=true, and raising the original exception
```
logging.error("An error occurred while fetching all available Prometheus metrics", exc_info=True)
raise e
```